prow/config/presubmits.yaml

```diff
@@ -2873,7 +2873,8 @@ presubmits:
           requests:
             cpu: "2"
             memory: 2Gi
-
+  
+  maistra/istio-images-centos:
```

Comment:
Can you move this up a little bit in the file? The bottom part of this file "belongs to" maistra-prow-testing org. IOW, put it together with the other maistra repos.